src/Microsoft.ML.Sweeper/Parameters.cs

```diff
@@ -555,7 +554,7 @@ public bool TryParseParameter(string paramValue, Type paramType, string paramNam
 
             // Extract the minimum, and the maximum value of the list of suggested sweeps.
             // Positive lookahead splitting at the '-' character.
-            // It is used for the Float and Long param types.
+            // It is used for the float and Long param types.
```

Comment:
>float [](start = 34, length = 5)

I think this is referring specifically to the `FloatParamOptions`, if you want to downcase it that's fine, but we should probably do the same with `long` here.